src/packages/@app/pages/WorkspaceExplorerPage/WorkspaceExplorerPage.svelte

```diff
@@ -1,38 +1,25 @@
 <script lang="ts">
-  import Input from "@library/forms/Input/Input.svelte";
-  import Information from "@library/icons/Information.svelte";
-  import Button from "@library/ui/button/Button.svelte";
-  import { SettingIcon } from "@library/icons";
-  import { notifications } from "@library/ui/toast/Toast";
-  import Tooltip from "@library/ui/tooltip/Tooltip.svelte";
-  import { WorkspaceSetting } from "@workspaces/features";
+  import { InviteToWorkspace, WorkspaceExplorer } from "@workspaces/features";
   import WorkspaceExplorerViewModel from "./WorkspaceExplorerPage.ViewModel";
   import { Modal } from "@library/ui";
+  import type { Observable } from "rxjs";
+  import { onDestroy } from "svelte";
   import { DeleteWorkspace } from "@common/features";
   import type { TeamDocument, WorkspaceDocument } from "@app/database/database";
 
   export let modifiedUser;
   export let collectionList;
   export let tab;
 
-  let showAbout = true;
-  let showWorkspaceSettings = false;
-  let workspaceName = tab.name;
-  let collectionLength = 0;
-  let workspaceDescription = tab.description;
-  let workspaceID = tab._data.path.workspaceId;
+  const _viewModel = new WorkspaceExplorerViewModel();
+  const activeWorkspace: Observable<WorkspaceDocument> =
+    _viewModel.activeWorkspace;
+
+  let isWorkspaceInviteModalOpen: boolean = false;
```

Comment:
i don't think so we need to give here type specifically as we have already set a default value.

Reply:
resolved